### ISSUE TITLE
"Couldn't fetch Telegram messages" error on startup

### ISSUE THREAD

**Issue by Andikki** [Suppen/TediCross#32]
After setting `skipOldMessages` setting to false, this error happens on startup:

```
Couldn't fetch Telegram messages. Reason: TypeError: Cannot read property 'send' of undefined
    at tgBot.on.wrapFunction (c:\Bots\TediCross\lib\telegram2discord\setup.js:195:61)
    at EventEmitter._.curry (c:\Bots\TediCross\lib\telegram2discord\setup.js:132:4)
    at EventEmitter.wrapper (c:\Bots\TediCross\node_modules\lodash\lodash.js:5232:19)
    at emitOne (events.js:115:13)
    at EventEmitter.emit (events.js:210:7)
    at updates.forEach (c:\Bots\TediCross\lib\telegram2discord\updategetter.js:87:15)
    at Array.forEach (<anonymous>)
    at bot.getUpdates.then (c:\Bots\TediCross\lib\telegram2discord\updategetter.js:69:12)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by Suppen** [comment]
Indeed it does... but after spitting out a few of those, it works as it should?

**Comment by Suppen** [comment]
The reason appears to be that the bot connects to Telegram faster than it connects to Discord, resulting in it trying to relay the messages before Discord is ready to receive them. Will find a fix

**Comment by Suppen** [comment]
Fixed in commit 2aff450cfa38adb7ae2da29455c0782f75f695b0
